src/Management/src/Tracing/TracingBaseServiceCollectionExtensions.cs

```diff
@@ -89,7 +89,7 @@ public static IServiceCollection AddDistributedTracing(this IServiceCollection s
                 var traceOpts = serviceProvider.GetRequiredService<ITracingOptions>();
                 ILogger logger = serviceProvider.GetService<ILoggerFactory>()?.CreateLogger("Steeltoe.Management.Tracing.Setup");
 
-                logger?.LogTrace("Found Zipkin exporter: {exportToZipkin}. Found Jaeger exporter: {exportToJaeger}. Found OTLP exporter: {exportToOtlp}.",
+                logger.LogTrace("Found Zipkin exporter: {exportToZipkin}. Found Jaeger exporter: {exportToJaeger}. Found OTLP exporter: {exportToOtlp}.",
```

Comment:
I suppose the call directly above this line should be `.GetRequiredService()`, to avoid a `NullReferenceException`.